python2.7-tensorflow-serving-gpu/Dockerfile

```diff
@@ -33,12 +33,6 @@ RUN cd /opt && \
     cd .. && \
     pip install numpy grpcio && \
     bazel build -c opt \
```

Comment:
Why don't we just pull from PyPI in these cases?

Reply:
Definitely should

Reply:
✅

Reply:
Sorry if this is a WIP, but you forgot the Python 2 images

Reply:
These are the tf-serving images, there are no PyPI packages for those

Reply:
Ah, got it!